nuxt.config.js

```diff
@@ -99,6 +99,7 @@ export default async () => {
             'nuxt-vue-multiselect',
             '@nuxtjs/google-gtag',
             '@nuxtjs/recaptcha',
+            '@nuxtjs/toast',
```

Comment:
I changed how we add the same `vue-toasted` library so that we can access it from Vuex (global) state configs (this necessary to work with existing architecture)